.github/workflows/wheel.yml

```diff
@@ -41,6 +40,15 @@ jobs:
             os: "macos-12"
           - os-arch: "macosx_arm64"
             os: "macos-12"
+          - cibw-python: "cp38"
+            python-version: "3.8"
+          - cibw-python: "cp39"
+            python-version: "3.9"
+          - cibw-python: "cp310"
+            python-version: "3.10"
+          - cibw-python: "cp311"
+            python-version: "3.11"
```

Comment:
You forget cp37 and Python 3.7 pair.
Or we can drop the support for Python 3.7 as its security support ended 8 month ago.
https://endoflife.date/pythong

Reply:
OK.
I'll remove Python 3.7.